user.go

```diff
@@ -31,12 +31,12 @@ type Officership struct {
 
 // Photo represents a photo of a user.
 type Photo struct {
-	PhotoId      uint   `json:"photoid"`
+	PhotoID      uint   `json:"photoid"`
```

Comment:
…here it feels like `PhotoID` should be `ID`, or `ID` in `Alias` and `Timeslot` should be `AliasID` and `TimeslotID`.